dex/order/order.go

```diff
@@ -218,6 +218,9 @@ func (p *Prefix) Time() int64 {
 // Time returns the order prefix's server time as a UNIX epoch time.
 func (p *Prefix) SetTime(t int64) {
 	p.ServerTime = time.Unix(t, 0).UTC()
+	// Force recomputation of the OrderID.
+	p.id = nil
+	p.uid = ""
```

Comment:
`panic` is a reasonable suggestion since the ID isn't valid until the server time is set.  I had this as a force recomputation setup though when there was a possibility that the server would re-write the time stamp.  That's not a thing anymore though.